budgetportal/tests/test_ppp_projects.py

```diff
@@ -0,0 +1,39 @@
+# !/usr/bin/python
+# -*- coding: utf-8 -*
+from datetime import datetime
+
+from django.test import override_settings
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+from budgetportal.models import (
+    InfrastructureProjectPart, FinancialYear)
+from budgetportal.tests.helpers import BaseSeleniumTestCase
+
+
+@override_settings(STATICFILES_STORAGE='django.contrib.staticfiles.storage.StaticFilesStorage')
```

Comment:
I think you won't need this if you run `docker-compose run app python manage.py collectstatic` - should this be part of the documentation or is it sufficient that it's in .travis.yml?

Please remove